src/vendorcode/dasharo/options.c

```diff
@@ -271,21 +274,21 @@ uint8_t cse_get_me_disable_mode(void)
 	}
 
 	/* Disable ME via HMRPFO if in Firmware Update Mode */
-	if (CONFIG(DASHARO_FIRMWARE_UPDATE_MODE) && fum) {
-		/* Check if already in HMRFPO mode */
-		if (cse_is_hfs1_com_secover_mei_msg())
-			return ME_MODE_DISABLE_HMRFPO;
-		else
-			disable_me_hmrfpo(&var);
-	}
+	//if (CONFIG(DASHARO_FIRMWARE_UPDATE_MODE) && fum) {
```

Comment:
That's quite radical, could have just added `CONFIG(DRIVERS_EFI_UPDATE_CAPSULES)` and a comment that locked IFD implies no HAP during capsule updates and we must avoid cold reset on enabling HMRFPO.  The check could even be more fine grained and additionally check for IFD being locked.  I just worry of some unexpected change in behaviour during updates on other platforms.

Reply:
Changed to disable HMRFPO if capsule update is enabled